File: 70-climbing-stairs/climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int total=0;
        if(n==1)return 1;
        if(n==2)return 2;
        int pre=1;
        int pre2=2;
        n-=2;
        while(n>0){
            total=pre+pre2;
            pre=pre2;
            pre2=total;
            n--;
        }
        return total;
    }
}

File: 70-climbing-stairs/climbing-stairs.c
int climbStairs(int n) {
	int i;
	int numb1;
	int numb2;
	i = 0;
	numb1 = 0;
	numb2 = 1;
	while(i != n)
	{
		if (i % 2 == 0)
			numb1 = numb1 + numb2;
		else
			numb2 = numb2 + numb1;
		i++;
	}
	if (i % 2 == 0)
			return numb2;
	return numb1;
}